packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.scss

```diff
@@ -7,17 +7,62 @@
 // --------------------------------------------------
 // Search box styles
 
-
-$SearchBox-field-padding-left: 10px;
-$SearchBox-height: 36px;
-
 .root {
   @include ms-font-m;
   @include ms-u-normalize;
   color: $ms-color-neutralPrimary;
   position: relative;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: stretch;
   margin-bottom: 10px;
   border: 1px solid $ms-color-themeTertiary;
+  height: 32px;
+}
+
+.icon {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+  color: $ms-color-neutralSecondaryAlt;
+  font-size: 16px;
+  flex-basis: 32px;
+  text-align: right;
```

Comment:
True true. It's a bit messy given the redlines I need to match. I'll see what I can do that'll flip with flex and not need direction.